src/python/ServerUtilities.py

```diff
@@ -248,3 +258,19 @@ def mostCommon(lst, default=0):
     except ValueError:
         return default
 
+
+def getTimeFromTaskname(taskname):
```

Comment:
Isn't task submission time in DataBase ? Relying on task name is ugly and fragile